### PR TITLE
Modified OS specific parts

### PR DESCRIPTION
**Pull request by sbluhm**
# Description

Adapting documentation to be include Enterprise Linux systems.

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes https://github.com/uyuni-project/uyuni/pull/4646

### CODE REVIEW

**Review by jcayouette — APPROVED** [review]

File: modules/specialized-guides/pages/salt/salt-formulas-custom.adoc [hunk 1]
```diff
@@ -14,6 +14,8 @@ RPM-based formulas must be placed in a specific directory structure to ensure th
 A formula contains two separate directories: [path]``states``, and [path]``metadata``.
 Folders in these directories need to have exactly matching names.
 
+Based on the operating system, the placeholder <serving directory> needs to be replaced to [path]``/srv`` for SUSE based systems or [path]``/var`` for Red Hat based systems.
```
Comment (jcayouette) [inline modules/specialized-guides/pages/salt/salt-formulas-custom.adoc:17]: "should be replaced with"

Comment (sbluhm) [reply modules/specialized-guides/pages/salt/salt-formulas-custom.adoc]: done

File: modules/specialized-guides/pages/salt/salt-formulas-custom.adoc [hunk 2]
```diff
@@ -14,6 +14,8 @@ RPM-based formulas must be placed in a specific directory structure to ensure th
 A formula contains two separate directories: [path]``states``, and [path]``metadata``.
 Folders in these directories need to have exactly matching names.
 
+Based on the operating system, the placeholder <serving directory> should be replaced with [path]``/srv`` for SUSE based systems or [path]``/var`` for Red Hat based systems.
```
Comment (keichwa) [inline modules/specialized-guides/pages/salt/salt-formulas-custom.adoc:17]:
```suggestion
Based on the operating system, the placeholder [literal]``<serving directory>`` should be replaced with [path]``srv`` for SUSE based systems or [path]``var`` for Red Hat based systems.
```

File: modules/specialized-guides/pages/salt/salt-formulas-custom.adoc [hunk 3]
```diff
@@ -33,18 +35,18 @@ For example, the [package]``locale-formula`` metadata directory is located in:
 If you have a custom formula that is not in an RPM, it must be in a state directory configured as a Salt file root.
 Custom state formula data must be in:
 ----
-/srv/salt/<custom-formula-name>/
+<serving directory>/salt/<custom-formula-name>/
```
Comment (keichwa) [inline modules/specialized-guides/pages/salt/salt-formulas-custom.adoc:38]:
```suggestion
/<serving directory>/salt/<custom-formula-name>/
```

File: modules/specialized-guides/pages/salt/salt-formulas-custom.adoc [hunk 4]
```diff
@@ -33,18 +35,18 @@ For example, the [package]``locale-formula`` metadata directory is located in:
...
 ----
 
 Custom metadata information must be in:
 ----
-/srv/formula_metadata/<custom-formula-name>/
+<serving directory>/formula_metadata/<custom-formula-name>/
```
Comment (keichwa) [inline modules/specialized-guides/pages/salt/salt-formulas-custom.adoc:43]:
```suggestion
/<serving directory>/formula_metadata/<custom-formula-name>/
```

File: modules/specialized-guides/pages/salt/salt-formulas-custom.adoc [hunk 5]
```diff
@@ -33,18 +35,18 @@ For example, the [package]``locale-formula`` metadata directory is located in:
...
 ----
 
 All custom folders must contain a [path]``form.yml`` file.
 These files are detected as form recipes and are applied to groups and systems from the {webui}:
 ----
-/srv/formula_metadata/<custom-formula-name>/form.yml
+<serving directory>/formula_metadata/<custom-formula-name>/form.yml
```
Comment (keichwa) [inline modules/specialized-guides/pages/salt/salt-formulas-custom.adoc:49]:
```suggestion
/<serving directory>/formula_metadata/<custom-formula-name>/form.yml
```

**Review by keichwa — APPROVED** [review]
thanks the PR!

For consistency, I'd remove the directory separator from the placeholder of the directory name.

**Review by 0rnela — APPROVED** [review]

**Comment by keichwa** [comment]
I think we should wait with merging until uyuni-project/uyuni#4646 is merged.

**Comment by sbluhm** [comment]
@keichwa on the one hand, I agree as they are connected. On the other hand it might as well be merged as it makes no difference to existing users. The different folders are a fact either way. I see #4646 as a bug fix really (and not a new feature).
So makes no real difference whether it is merged now or later.

**Comment by keichwa** [comment]
> @keichwa on the one hand, I agree as they are connected. On the other hand it might as well be merged as it makes no difference to existing users. The different folders are a fact either way. I see #4646 as a bug fix really (and not a new feature). So makes no real difference whether it is merged now or later.

Thanks for your explanation! You are right, we can merge it now.  If nobody vetoes, I'll do it tomorrow; of course, all you others could do it as well.  We probably should this also backport to 4.2 documentation.

**Pull request merged**